tasks/e2e/cypress/integration/01-tutorial/tutorial.spec.js

```diff
@@ -41,30 +41,43 @@ describe('The Redwood Tutorial - Golden path edition', () => {
 
   it('1. Our First Page', () => {
     //redwoodjs.com/tutorial/our-first-page
+    cy.task('execa', {
+      cmd: 'yarn redwood generate page home / --force',
+      cwd: BASE_DIR,
+    });
     cy.visit('http://localhost:8910')
-    cy.exec(`cd ${BASE_DIR}; yarn redwood generate page home / --force`)
     cy.get('h1').should('contain', 'HomePage')
   })
 
   it('2. A Second Page and a Link', () => {
     // https://redwoodjs.com/tutorial/a-second-page-and-a-link
-    cy.exec(`cd ${BASE_DIR}; yarn redwood generate page about --force`)
+    cy.task('execa', {
+      cmd: 'yarn redwood generate page about --force',
+      cwd: BASE_DIR,
+    });
+    cy.wait(15000)
```

Comment:
Even with 15 seconds it doesn't always work. But it works more often than when it's 5 seconds